lib/sailthru.js

```diff
@@ -160,10 +169,11 @@
   SailthruClient = (function() {
     SailthruClient.logging = true;
 
-    function SailthruClient(api_key, api_secret, api_url) {
+    function SailthruClient(api_key, api_secret, proxy_url, api_url) {
```

Comment:
doesn't order matter here? anyone that might be using what was the third argument (`api_url`) would be affected by this.

Reply:
Yeah, this is true, and its something Jack and I discussed. `api_url` is entirely undocumented, and so its entirely unlikely to have any effect outside of a handful of dev environments. This is a use case we'd want to discuss as we're considering a re-write but for now the effects seem so minute that we were okay to move forward with it. I placed `proxy_url` there so that we could add it to our existing documentation without exposing `api_url`. Its a small fix if we decide its a problem later on.